streams/bo.m3u

```diff
@@ -83,3 +81,21 @@ https://solo.disfrutaenlared.com:1936/teleestrella/teleestrella/playlist.m3u8
 https://video.live.com.bo:3699/live/megavisionlive.m3u8
 #EXTINF:-1 tvg-id="UnivalleTelevision.bo",Univalle Televisión (1080p) 
 https://master.tucableip.com/univalletv/playlist.m3u8
+#EXTINF:-1 tvg-id="AgendaMineraTV.bo",Agenda Minera TV (720p) 
+https://srv.panelcast.net/amitel/amitel/playlist.m3u8
+#EXTINF:-1 tvg-id="DeltaTV.bo",Delta TV (480p) [Not 24/7]
+https://vsrv1.az-streamingserver.com:3068/live/vdsocmnulive.m3u8
+#EXTINF:-1 tvg-id="UnitelSantaCruz.bo",Unitel Santa Cruz (1080p) 
+https://cdn.digital.com.bo/__cl/cg:sworigin2/__c/UNITEL/__op/hls-default/__f/index.m3u8
```

Comment:
Does not work outside of Bolivia it seems.

Reply:
Working for me right now in Spain. Pretty sure is Not 24/7.

Reply:
![image](https://github.com/iptv-org/iptv/assets/30985701/9ac21b01-c660-4432-bbb0-34fc73187a11)
Doesn't seem to work either on Streamtest.

Reply:
Still working for me :/

Reply:
Still working for me :/
https://i.imgur.com/FcJh2uy.jpg

Reply:
I guess then it's a lock for HISPAM